templates/desktop/index.html

```diff
@@ -1,98 +1,94 @@
-{% if ubuntu_kylin %}
-<div class="row">
-    <div class="six-col">
-        <div class="twelve-col">
-            {% include "templates/_cms-block.html" with block_content=ubuntu_kylin extra_classes="ubuntu-kylin cta-links" %}
-        </div>
-        {% include "templates/_cms-block.html" with block_content=ubuntu_kylin_additional extra_classes="ubuntu-kylin-additional" %}
+    <div class="col-4 p-divider__block">
+      <h4>对于企业用户</h4>
+      <p>安全、直观且易于管理，可节约企业成本费用，但丝毫不影响其性能；可以大大降低企业IT硬件和运营维护的总成本。</p>
+      <a class="p-link--external" href="maito:chinamarketing@canonical.com">联络我们获得更多信息</a>
     </div>
-    <div class="six-col last-col">
-        <img src="{{ STATIC_URL }}img/homepage/dell-xps-laptop-front.png" alt="Kylin on an XPS laptop">
+    <div class="col-4 p-divider__block">
+      <h4>对于开发者</h4>
+      <p>直观、易用、安全可靠。 触手可得广泛的开发工具、强大的开发者社区、以及部署应用程序的最快方法。是工程师的必备系统。目前可以直接运行于Windows桌面系统。</p>
+      <a href="http://cn.ubuntu.com/download" class="p-button">下载</a>
+      <a class="p-link--external" href="去Ubuntu开发者网站看看"></a>
     </div>
-</div>
-{% endif %}
+  </div>
+</section>
 
-{# ================ Who is behind Ubuntu? ================ #}
-{% if who_is_behind_kylin %}
-<div id="ubuntu-kylin-backers" class="row">
-    <div class="six-col">
-        {% include "templates/_cms-block.html" with block_content=who_is_behind_kylin %}
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-6">
+      <h3>Ubuntu Kylin</h3>
+      <p>优麒麟(英文名为Ubuntu Kylin)是基于Ubuntu的一款官方衍生版。 它是一款专门为中国市场打造的免费操作系统，而且它已经被录入中国政府采购条例名单中。它包括Ubuntu用户期待的各种功能，并配有必备的中文软件及程序。</p>
+      <a href="http://cn.ubuntu.com/download" class="p-button">下载Ubuntu</a>
+      <a href="http://www.ubuntukylin.com/" class="p-link--external"></a>
     </div>
-    <div class="six-col text-center last-col">
-            <ul class="no-bullets clearfix align-center">
-                <li><img src="{{ STATIC_URL }}img/desktop/logo-canonical-288.png" alt="Canonical" width="288" height="40"></li>
-                <li><img src="{{ STATIC_URL }}img/desktop/logo-csip-180.png" alt="CSIP" width="180" height="71"></li>
-                <li><img src="{{ STATIC_URL }}img/desktop/logo-nudt-288.png" alt="NUDT" width="288" height="72"></li>
-            </ul>
-        </div>
-</div>
-{% endif %}
+    <div class="col-6">
+      <img src="{{ STATIC_URL }}img/homepage/dell-xps-laptop-front.png" alt="Kylin on an XPS laptop">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-6">
+    <h3>优麒麟是由</h3>
+    <p><a class="p-link--external" href="http://www.canonical.com">Canonical</a>，<a class="p-link--external" href="http://www.csip.org.cn">工业和信息化部软件与集成电路促进中心（CSIP）</a>以及<a class="p-link--external" href="http://www.nudt.edu.cn">国防科学技术大学（NUDT）</a>携手打造。
+    目前，开发团队正在与开源社区合作打造Ubuntu的核心功能，以更好地适应中国用户。 除了Ubuntu软件中心推出的本土化版本以外，优麒麟还带最常用的中文程序。</p>
+  </div>
+    <div class="col-6">
+    <ul class="p-list u-align--center">
```

Comment:
Should use the inline-images pattern here: https://docs.vanillaframework.io/en/patterns/inline-images